sdk/Lusid.Sdk.Tests/tutorials/Ibor/Properties.cs

```diff
@@ -148,5 +150,74 @@ public void Create_Portfolio_With_Metric_Property()
             Assert.That(metricProperty.Value.MetricValue.Value, Is.EqualTo(metricPropertyValueRequest.MetricValue.Value));
             Assert.That(metricProperty.Value.MetricValue.Unit, Is.EqualTo(metricPropertyValueRequest.MetricValue.Unit));
         }
+        
+        [Test]
+        public void Create_Portfolio_With_MultiValue_Property()
+        {
+            var uuid = Guid.NewGuid().ToString();
+            var propertyName = $"manager-{uuid}";
+            var effectiveDate = new DateTimeOffset(2018, 1, 1, 0, 0, 0, TimeSpan.Zero);
+            
+            //    Details of the property to be created
+            var multiValuePropertyDefinition = new CreatePropertyDefinitionRequest(
+                
+                //    The domain the property is to be applied to
+                domain: CreatePropertyDefinitionRequest.DomainEnum.Portfolio,
+                
+                //    The scope the property will be created in
+                scope: TestDataUtilities.TutorialScope,
+                
+                //    When the property value is set it will be valid forever and cannot be changed.
+                //    Properties whose values can change over time should be created with LifeTimeEnum.TIMEVARIANT
+                lifeTime: CreatePropertyDefinitionRequest.LifeTimeEnum.Perpetual,
+                
+                // Multi-value properties have constraint style "Collection"
+                constraintStyle: "Collection",
+                
+                code: propertyName,
+                valueRequired: false,
+                displayName: "Manager",
```

Comment:
Same as above, plural might be a better example.

Reply:
Agree, changed.